compiler/src/iree/compiler/Codegen/LLVMGPU/TransformExtensions/LLVMGPUExtensions.cpp

```diff
@@ -655,5 +656,33 @@ transform_dialect::VectorToMMAConversionOp::applyToOne(
   return DiagnosedSilenceableFailure::success();
 }
 
+DiagnosedSilenceableFailure transform_dialect::PromoteOperandsOp::applyToOne(
+    Operation *target, transform::ApplyToEachResultList &results,
+    transform::TransformState &state) {
+  OpBuilder builder(getContext());
+  OpBuilder::InsertionGuard g(builder);
+  builder.setInsertionPoint(target);
+  SmallVector<int64_t> indices = llvm::to_vector(getIndices());
+  auto numOperands = target->getNumOperands();
+  bufferization::BufferizationOptions options;
+  for (int64_t index : indices) {
+    if ((index >= 0) && (index < numOperands)) {
+      FailureOr<Value> ret = bufferization::allocateTensorForShapedValue(
+          builder, target->getLoc(), target->getOperand(index), false, options,
+          true);
+      if (failed(ret)) {
+        target->emitOpError("failed to promote operand");
```

Comment:
I think you want:
`return emitDefaultDefiniteFailure(target) << "failed to promote operand";` same below.

Reply:
thanks will change